pkg/ebpf/tracer.go

```diff
@@ -336,6 +337,7 @@ func (t *Tracer) GetActiveConnections(clientID string) (*Connections, error) {
 	}
 
 	conns := t.state.Connections(clientID, latestTime, latestConns)
+	t.determineConnectionIntraHost(conns)
```

Comment:
nit, does this need to be done in the tracer or can it be done in `Connections` ?

Reply:
Does this actually fit better in `state.Connections` ? `tracer` is already doing a bunch of post-processing (like  handling connection direction, ip translations, and DNS). This seems to fit in nicely.

Reply:
🤷‍♂ my thought was that since the method does nothing specific with the tracer data (the method itself is stateless) it would be better off elsewhere so that other calls to `Connections` would have the data.  

I don't have strong feelings though.

Reply:
I've been playing around with this, and it can be done in `Connections` fairly simply. I have the changes on my local branch and will push them up.